dlib/test/images/README.txt

```diff
@@ -0,0 +1,5 @@
+This directory contains images that are problematic to parse,
+for instance because they contain errors.
+
+They are here, to be able to prove bugs, which may have been fixed.
```

Comment:
Rename `dlib/test/images/problematic/integer_overflow_3d4e1dc8.txt ` to something other than a `.txt` file